app/src/app/components/specmath-modal/modal.module.ts

```diff
@@ -48,6 +50,8 @@ import { DefaultsFileUploadComponent } from './defaults-file-upload/defaults-fil
     ModalComponent,
     SpecNameInputComponent,
     DefaultsFileUploadComponent,
+    SpecFilesUploadComponent,
+    MatTooltipModule
   ],
   providers: [],
```

Comment:
nit: don't need this line since it is empty

Reply:
Updated